container_files/assets/www/doc/builtin/procedures/MeltProcedure.md

```diff
@@ -0,0 +1,61 @@
+# Melt Procedure
+
+This procedure type allows you to perform a melt operation on a dataset. Melting 
+a dataset takes a set of columns to keep fixed and a set of columns to melt and
+creates a new rows, one per column to melt, while copying as is the columns to 
+keep fixed.
+
+An example usage is importing JSON data where some fields contain an array of 
+objects and the way we want to process it is one object per row.
+
+## Configuration
+
+![](%%config procedure melt)
+
+## Example
+
+Suppose the following dataset `data` with the `friends` column containing strings.
+
+| *rowName* | *name* | *age* | *friends* |
+|-----------|--------|-------|-----------|
+| row1 | bill | 25 | [{"name": "mich", "age": 20}, {"name": "jean", "age": 18}] |
```

Comment:
i would give this example with an atomic JSON list like `['mich', 'jean']` instead of a list-of-objects